docs/builders/webpack.md

```diff
@@ -128,6 +128,19 @@ Storybook's default Webpack configuration provides support for most project setu
 
 <!-- prettier-ignore-end -->
 
+While if you only want to add an import path, like `@` import which is used in `NextJS 13` default setup, you can use the following solution without adding a package:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/storybook-main-ts-module-resolution-atsign-import.js.mdx',
+    'common/storybook-main-ts-module-resolution-atsign-import.ts.mdx',
+  ]}****
+/>
+
+<!-- prettier-ignore-end -->
```

Comment:
@noviceGuru we have a bit of room for improvement here, and we can adjust this to the following:
```md
However, if you're working with a framework that provides a default aliasing configuration (e.g., Next.js, Nuxt) and you want to configure Storybook to use the same aliases, you may not need to install any additional packages. Instead, you can extend the default configuration of Storybook to use the same aliases provided by the framework. For example, to set up an alias for the `@` import path, you can add the following to your `.storybook/main.js|ts` file:

<!-- prettier-ignore-start -->

<CodeSnippets
  paths={[
    'common/storybook-main-ts-module-resolution-atsign-import.js.mdx',
    'common/storybook-main-ts-module-resolution-atsign-import.ts.mdx',
  ]}
/>

<!-- prettier-ignore-end -->

```